docs/_pages/features.md

```diff
@@ -40,7 +40,7 @@ It is required that the 1st run is done with an internet connection, this flag w
 ## Stub (domain) versions
 The core will store the last few versions of stub files generated in a cache folder to compare future changes:
 ```
-.openapi-nodegen/cache/compare
+.generate-it/cache/compare
```

Comment:
whoops - I did a find and replace fail here. This should still be `.openapi-nodegen`

Reply:
Fixed!

Reply:
Fixed